Amazon.Ion.ObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -109,6 +109,24 @@ public void SerializesAndDeserializesCustomPropertyNames()
             Check(TestObjects.fmRadio);
         }
 
+        [TestMethod]
+        public void SerializesAndDeserializesGetterAndSetterMethods()
+        {
+            var serializer = new IonSerializer();
+            var ruler = new Ruler {size = 20};
+
+            var stream = serializer.Serialize(ruler);
+            IIonStruct serialized = StreamToIonValue(stream);
+            Assert.IsTrue(serialized.ContainsField("size"));
+            
+            // Note that GetSize() returns size + 10.
```

Comment:
The comment doesn't actually mean anything here, since you're not testing the values directly. If that's the intent, you should probably check against the specific value.